src/main/views/home.njk

```diff
@@ -13,40 +13,37 @@ Find a court or tribunal - GOV.UK
       {% if serviceName %} {{ serviceName }} {% endif %}
     </h1>
 
-    <p class="govuk-body-m">Use this service to find a court or tribunal in England and Wales</p>
+    <p class="govuk-body-m">{{ p1 }}</p>
 
     <ul class="govuk-list govuk-list--bullet">
 
-      <li>address</li>
-      <li>contact details</li>
-      <li>opening times</li>
-      <li>building information e.g. disabled access or parking</li>
-      <li>to help me get an update</li>
+      {% for bullet in bullets %}
+      <li>{{ bullet }}</li>
+      {% endfor %}
 
     </ul>
 
     {{ govukButton({
-    text: "Start now",
+    text: button,
```

Comment:
i assume button is a variable, does it not need to be {{}} or {%= %} for nunjucks? i could be wrong.

Reply:
It's already in {{ govukButton .... }}